services/provider/proto/provider.proto

```diff
@@ -169,6 +169,7 @@ message ReportStatusRequest{
 }
 
 message ReportStatusResponse{
-    // Contains subscription channel of provider operator for client operator to match
-    string desiredClientOperatorChannel = 1;
```

Comment:
Probably No.1 rule in Proto, do not reuse field tags, reserve if old tags are being deprecated.

We have one relaxation that atm we need to support clients lagging by 1 minor version and this update breaks the compatibility.